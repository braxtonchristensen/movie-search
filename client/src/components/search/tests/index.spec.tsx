import React from 'react';
import renderer from 'react-test-renderer';
import Search from '../index';

describe('<Search />', () => {
  describe('snapshots', () => {
    it('should match snapshot', () => {
      const rendered = renderer.create(
        <Search />
      ).toJSON();

      expect(rendered).toMatchSnapshot();
    });
  });
});
