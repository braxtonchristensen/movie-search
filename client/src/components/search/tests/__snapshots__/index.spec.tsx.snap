// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> snapshots should match snapshot 1`] = `
<div
  className="sc-bdVaJa kwuFdg"
>
  <input
    className="sc-bwzfXH gJpQtX"
    placeholder="Search"
    type="text"
  />
</div>
`;
